ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -100,6 +107,12 @@ internal class PaywallViewModelImpl(
         }
     }
 
+    override fun openURL(url: URL) {
+        // TODO: implement
+//        val intent = Intent(Intent.ACTION_VIEW, url.toURI())
+//        startActivity(intent)
```

Comment:
@tonidero @vegaro need help with this 😬

Reply:
Looking at the code, I think you might need to pass in the activity context (From `LocalContext.current` in a composable function) and also need to pass a `Uri`, not an `URI` (I know, very confusing). So change the code to something like:
```
val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url.toString()))
context.startActivity(intent)
```
Lmk how that goes!

Reply:
Thanks! Let me try this

Reply:
It works great! 🎉